### DISCUSSION TITLE
HA 2024.6.x Ring MQTT

### DISCUSSION THREAD

**Discussion by dmkjr** [tsightler/ring-mqtt#843] [Support Q&A]
Anyone else tried out the current 6.x beta from HA?  My Ring MQTT seems either I’m missing something, or it isn’t working. 

![IMG_3571](https://github.com/tsightler/ring-mqtt/assets/662675/327c2ade-53dd-4c28-a764-99bde7090280)

I try and stay up to date, but perhaps I missed something?

**Comment by tsightler** [comment]
Well, being on beta/RC version is not "staying updated" it is "living on the bleeding edge", so expect to get cut even more often than with normal HA, which is already an unstable mess of a platform IMO (I'm a big believer that home automation should be near 100% functional and monthly breaking updates flies in the face of this principal for me).

That being said, the logs above do not appear to be from ring-mqtt at all, but rather are from Home Assistant, so there's likely nothing to do for ring-mqtt anyway as ring-mqtt, as it's name implies, has nothing to do with HA, it only sends messages via MQTT, thus, unless it's yet another unannounced breaking change from Home Assistant's MQTT integration, then it's almost certainly an issue with HA.  One second of Google turned up a similar issues reported just a few hours ago for a completely different alarm configuration:

https://github.com/home-assistant/core/issues/118668

**Reply by dmkjr** [reply]
Ahh, I did search but didn't even think about this not being an issue from the MQTT side. It was late but I definitely see the information from the logs in the Docker container. HA, not seeing much outside of the displayed error. 

I've been messing with HA for years. While the updates and breaks get frustrating, it keeps me occupied in my down time.

**Reply by tsightler** [reply]
Down time must be nice, if I had any I might enjoy playing with it too, as it is I just find it an essential part of my home so I'm as frustrated when it doesn't work as when one of my appliances doesn't work, although at least it's usually easy to fix and doesn't costs anything but my time.  I just wish they had a slower cycle.  It hasn't been as bad in the last few years though, feels much more stable now than 4-5 years ago.

**Reply by dmkjr** [reply]
My wife doesn't allow me to have it too "essential".  Sure, I can control most things or have shoehorned something together, but if the garage button didn't work because I put a smart switch on there, I'd be in the dog house. So my "changes" have to be wife approved.

**Reply by MasterDevwi** [reply]
I just updated to 2024.6.0 and noticed the same issue. The the service call fails unless I enter a code. Strangely it'll succeed with _any_ four-digit code. It doesn't have to be one set up in Ring. So I suspect there's a larger issue in HA.
